src/udp.c

```diff
@@ -108,21 +109,21 @@ static void _trudpUdpSetNonblock(int fd) {
  * @param server
  */
 static void _trudpUdpHostToIp(struct sockaddr_in *remaddr, const char *server) {
-    int result = inet_pton(AF_INET, server, &remaddr->sin_addr);
-
-    if (result != -1) {
-        /* When passing the host name of the server as a */
-        /* parameter to this program, use the gethostbyname() */
-        /* function to retrieve the address of the host server. */
-        /***************************************************/
-        /* get host address */
-        struct hostent *hostp = gethostbyname(server);
-        if (hostp == NULL) {
-            // ...
-        } else {
-            memcpy(&remaddr->sin_addr, hostp->h_addr_list[0], sizeof(remaddr->sin_addr));
-        }
-    }
+    // int result = inet_pton(AF_INET, server, &remaddr->sin_addr);
```

Comment:
Thats pretty confusing.
Having function that should convert one representation to another, but doing nothing - "happy debugging, bi###s!"